test/Interop/Cxx/class/inline-function-codegen/method-calls-function-execution.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop -Xcc -std=c++20 -Xfrontend -disable-availability-checking)
```

Comment:
nit: since you're already changing this line, maybe use `-cxx-interoperability-mode=default` rather than `-enable-experimental-cxx-interop`?